src/App.js

```diff
@@ -1,8 +1,31 @@
 import './App.css';
+import { Counter } from './Components/Counter/Counter';
+import { SearchForm } from './Components/SearchForm/SearchForm';
+import { GenreSelector } from './Components/GenreSelector/GenreSelector'
 
 function App() {
+
+  const onSearch = function (input) {
+    console.log(input);
+  }
+
+  const onSelectGenre = function (genre) {
+    console.log(genre)
+  }
+
+  const genreList = [
```

Comment:
Well, I'd recommend moving the genreList out of the component, as it is recreated on every component re-render.
And looks like it's not gonna change during the runtime, so we can keep in even in the separate file (or at least out of the component)